python/tvm/relay/build_module.py

```diff
@@ -129,108 +103,203 @@ def build_config(**kwargs):
     return BuildConfig(**kwargs)
 
 
-def _bind_params_by_name(func, params):
-    """Bind parameters of function by its name."""
-    name_dict = {}
-    for arg in func.params:
-        name = arg.name_hint
-        if name in name_dict:
-            name_dict[name] = None
-        else:
-            name_dict[name] = arg
-    bind_dict = {}
-    for k, v in params.items():
-        if k not in name_dict:
-            continue
-        arg = name_dict[k]
-        if arg is None:
-            raise ValueError("Multiple args in the function have name %s" % k)
-        bind_dict[arg] = _expr.const(v)
-    return _expr.bind(func, bind_dict)
-
-
-def optimize(func, target=None, params=None):
```

Comment:
`optimize` is used in https://github.com/dmlc/tvm/blob/6a4d71ff40915611bd42b62994992b879e6be610/python/tvm/relay/quantize/quantize.py#L300
can you fix it?
I noticed that for some reason e2e quantization test is disabled.

Reply:
@vinx13 Thanks for pointing out. For quantization test, it was tracked here by #2718 

I can probably enable it in a different PR.

Reply:
I ended up enabling the quantization tests in this PR as we need to update the `optimize` function here. I will keep my eye on it and ping @ZihengJiang once it's flaky again.